### PR TITLE
[refactor-696]Changing imports in atoms and molecules

### LINKED ISSUE #696: [Refactor] Absolute imports
Support Absolute imports
change imports to use Absolute imports (same folder relative is fine: `./MyComponent`)

Resources:
https://dev.to/mr_frontend/absolute-imports-in-create-react-app-3ge8

you might also consider these changes (only if lint/vscode issues occur)
https://medium.com/hackernoon/absolute-imports-with-create-react-app-4c6cfb66c35d

### PR DESCRIPTION
**Pull request by YaronAvraham93**
close #696

### CODE REVIEW

File: src/pages/HomePage.tsx [hunk 1]
```diff
@@ -1,15 +1,15 @@
 import React, { FC, useEffect } from 'react';
 import WidgetsTemplate from '../components/organisms/WidgetsTemplate';
```
Comment (daniel-ilan) [inline src/pages/HomePage.tsx:2]: Shouldn't this also be imported from an absolute path?

File: src/components/organisms/News.tsx [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import React, { FC } from 'react';
 import { Link, Typography } from '../atoms';
```
Comment (daniel-ilan) [inline src/components/organisms/News.tsx:2]: Shouldn't this also be imported from an absolute path ?

File: src/components/organisms/Footer.tsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React, { FC, useState } from 'react';
 import { useTranslation } from 'react-i18next';
-import logoHasdna from '../../assets/hasadna.png';
+import logoHasdna from 'assets/hasadna.png';
 import { makeStyles } from '@material-ui/core/styles';
 import { Box } from '@material-ui/core';
 import { Typography } from '../atoms';
```
Comment (daniel-ilan) [inline src/components/organisms/Footer.tsx:6]: Shouldn't this also be imported from an absolute path ?

File: src/components/molecules/widgets/MostSevereAccidentsTableWidget.tsx [hunk 4]
```diff
@@ -1,8 +1,8 @@
 import React, { FC } from 'react';
 import { useTranslation } from 'react-i18next';
 import TableView from '../Table/TableView';
-import { IWidgetMostSevereAccidentsTableData } from '../../../models/WidgetData';
-import { toJsDateFormat } from '../../../utils/time.utils';
+import { IWidgetMostSevereAccidentsTableData } from 'models/WidgetData';
+import { toJsDateFormat } from 'utils/time.utils';
 import { createTableData } from '../Table/formatTableData.service';
```
Comment (daniel-ilan) [inline src/components/molecules/widgets/MostSevereAccidentsTableWidget.tsx:6]: Shouldn't this be imported as an absolute path?

File: src/components/molecules/widgets/MostSevereAccidentsTableWidget.tsx [hunk 5]
```diff
@@ -1,8 +1,8 @@
 import React, { FC } from 'react';
 import { useTranslation } from 'react-i18next';
 import TableView from '../Table/TableView';
```
Comment (daniel-ilan) [inline src/components/molecules/widgets/MostSevereAccidentsTableWidget.tsx:3]: Shouldn't this be imported as an absolute path?

**Review by SejoB — COMMENTED** [review]
fix requested changes of @daniel-ilan

**Review by daniel-ilan — APPROVED** [review]

**Review by IsaacM890 — APPROVED** [review]

**Pull request merged**